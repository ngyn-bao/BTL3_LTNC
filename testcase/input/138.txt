INSERT x number
BEGIN
INSERT x string
PRINT
END