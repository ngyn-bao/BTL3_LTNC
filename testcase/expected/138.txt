success
success
x//1